JFRWebSocket.m

```diff
@@ -59,13 +59,13 @@ @interface JFRWebSocket ()<NSStreamDelegate>
 @property(nonatomic, strong)NSInputStream *inputStream;
 @property(nonatomic, strong)NSOutputStream *outputStream;
 @property(nonatomic, strong)NSOperationQueue *writeQueue;
-@property(nonatomic, assign)BOOL isRunLoop;
 @property(nonatomic, strong)NSMutableArray *readStack;
 @property(nonatomic, strong)NSMutableArray *inputQueue;
 @property(nonatomic, strong)NSData *fragBuffer;
 @property(nonatomic, strong)NSMutableDictionary *headers;
 @property(nonatomic, strong)NSArray *optProtocols;
 @property(nonatomic, assign)BOOL isCreated;
+@property(nonatomic)dispatch_queue_t callbackQueue;
```

Comment:
Could you name this something less likely to be confused with the the delegate callback queue? Perhaps "networkQueue" or "workQueue"?